client/rpcserver/rpcserver.go

```diff
@@ -36,9 +36,12 @@ const (
 	// is closed.
 	rpcTimeoutSeconds = 10
 
-	// RPC version
+	// RPC version. Move major up one for breaking changes. Move minor for
+	// backwards compatible features. Move patch for bug fixes. It is
+	// intended that this always match dexcctl's version, so also update
+	// there accordingly.
 	rpcSemverMajor = 0
-	rpcSemverMinor = 0
+	rpcSemverMinor = 4
 	rpcSemverPatch = 0
 )
```

Comment:
Am I using these wrong? I guess it should be 4 if dexcctl was at 3.

Reply:
dexcctl was for the product like v0.3.  I think these `rpcSemver*` vars represent the RPC protocol version.  I think we should bump these RPC versions independently of product, like the dcrd/dcrwallet RPC versions do.  So, we can bump this to `rpcSemverMinor = 1` I think.

Reply:
What I mean is that we probably should be at `rpcSemverMinor = 20` by now, but we haven't bothered.  (Is even there any communication of the version to RPC clients to make this useful yet?)

Reply:
I will set it to minor 1.